src/ui/Controllers/CourseController.cs

```diff
@@ -278,7 +336,8 @@ private VariantViewModel LoadViewModel(UserOrganisation org, Course course, bool
                 LiveSearchUrl = searchAndCompareUrlService.GetCoursePageUri(org.UcasCode, courseVariant.ProgrammeCode),
                 AllowPreview = featureFlags.ShowCoursePreview,
                 AllowPublish = featureFlags.ShowCoursePublish,
-                AllowLiveView = featureFlags.ShowCourseLiveView
+                AllowLiveView = featureFlags.ShowCourseLiveView,
+                IsSalary = course.ProgramType.Equals("SS", StringComparison.InvariantCultureIgnoreCase)
```

Comment:
This is the wrong place for this logic, though I think we might be worrying about that later.

Reply:
Agreed wrong place as there is limited right place for it